.skypageslocales/resources_en_US.json

```diff
@@ -0,0 +1 @@
+{"stache_copyright_label":{"message":"Blackbaud, Inc. All rights reserved.","_description":"Text to accompany the copyright date in the footer"},"stache_sidebar_label":{"message":"Sidebar Navigation","_description":"Label for the sidebar navigation"},"stache_sidebar_toggle_button":{"message":"Toggle sidebar","_description":"Button text to show or hide sidebar menu"}}
```

Comment:
Sorry, I added this file to my branch's gitignore; the temp locales shouldn't be committed back.